src/libLookup/Lookup.cpp

```diff
@@ -1508,9 +1508,7 @@ bool Lookup::ProcessSetTxBodyFromSeed(const vector<unsigned char>& message,
 
     unique_lock<mutex> lock(m_mutexSetTxBodyFromSeed);
 
-    if (IsMessageSizeInappropriate(message.size(), offset,
-                                   TRAN_HASH_SIZE
-                                       + Transaction::GetSerializedSize()))
```

Comment:
Instead of removing the GetSerializedSize, is it possible to put the unlimited data type at the end of Serialization, and try to verify whether those fixed-length data has met the requirement? You can create a new function if feel easy to do so.